mslib/msui/msui.py

```diff
@@ -1056,6 +1056,7 @@ def closeEvent(self, event):
             if self.config_editor is not None:
                 self.config_editor.restart_on_save = False
                 self.config_editor.close()
+                from PyQt5 import QtTest
```

Comment:
we need to replace QtTest by something else. This loads and needs also a framerwork installed which usually is only needed for testing Qt.
slide 13
https://www.slideshare.net/ICSinc/qt-test-framework

Reply:
Probably, this could be replaced.